src/critnib/critnib.c

```diff
@@ -137,10 +138,12 @@ struct critnib {
  * atomic load
  */
 static void load(void *src, void *dst) {
+    utils_annotate_acquire(src);
```

Comment:
actually, could you move the annotations to `utils_atomic_*` functions?

Reply:
Done